client/my-sites/people/edit-team-member-form/edit-user-form.jsx

```diff
@@ -130,12 +159,12 @@ class EditUserForm extends React.Component {
 		if ( true === changedSettings.isExternalContributor ) {
 			requestExternalContributorsAddition(
 				siteId,
-				undefined !== user.linked_user_ID ? user.linked_user_ID : user.ID
+				hasWPCOMAccountLinked ? user?.linked_user_ID || user.ID : user.ID // On simple sites linked_user_ID is undefined for connected users.
```

Comment:
Nitpick: Found a bit hard to understand this. I think the code below should work too and looks simpler?
```suggestion
				user?.linked_user_ID ?? user?.ID
```